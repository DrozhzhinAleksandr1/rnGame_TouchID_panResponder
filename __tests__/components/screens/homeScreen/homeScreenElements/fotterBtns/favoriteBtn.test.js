import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { FavoriteBtn } from '../../../../../../app/components/screens/homeScreen/homeScreenElement/fotterbtns/favoriteBtn';

it('create snapshot FavoriteBtn', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<FavoriteBtn isQuotesFavorite={true} setIsQuotesFavorite={()=>{}} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});