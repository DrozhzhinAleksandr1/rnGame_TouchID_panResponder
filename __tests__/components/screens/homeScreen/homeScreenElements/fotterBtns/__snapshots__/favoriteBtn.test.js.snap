// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot FavoriteBtn 1`] = `
<TouchableWithoutFeedback
  onPress={[Function]}
>
  <Component
    style={Object {}}
  >
    <StarGreen />
  </Component>
</TouchableWithoutFeedback>
`;
